tests/cdi/cdi_test.go

```diff
@@ -0,0 +1,53 @@
+package cdi_test
+
+import (
+	"os"
+	"flag"
+
+	. "github.com/onsi/ginkgo"
+	"kubevirt.io/kubevirt-ansible/tests"
+	ktests "kubevirt.io/kubevirt/tests"
+)
+
+// template parameters
+const (
+	pvcEPHTTPNOAUTHURL = "https://download.cirros-cloud.net/0.4.0/cirros-0.4.0-x86_64-disk.img"
+	pvcName            = "golden-pvc"
+	vmName             = "test-vm"
+	vmAPIVersion       = "kubevirt.io/v1alpha2"
+	rawPVCFilePath     = "tests/manifests/golden-pvc.yml"
+	rawVMFilePath      = "tests/manifests/test-vm.yml"
+	sepP               = "-p"
+)
+
+var _ = Describe("Importing and starting a VM using CDI", func() {
+	flag.Parse()
+	ktests.SkipIfNoOc()
+	dstPVCFilePath := "/tmp/test-pvc"
+	dstVMFilePath := "/tmp/test-vm"
+
+	url, ok := os.LookupEnv("STREAM_IMAGE_URL")
```

Comment:
Is this env variable documented anywhere? IOW how does someone running this test know about this var?